_articles/tr/how-to-contribute.md

```diff
@@ -0,0 +1,444 @@
+---
+lang: tr
+title: Açık Kaynağa Nasıl Katkıda Bulunulur
+description: Açık kaynağa katkıda bulunmak ister misiniz? İlk defa çalışanlar ve tecrübeliler
+  için açık kaynak katkıları yapma rehberi.
+class: contribute
+toc:
+  why-contribute-to-open-source: Açık kaynağa neden katkıda bulunur?
+  what-it-means-to-contribute: Katkıda bulunmak ne demektir?
+  orienting-yourself-to-a-new-project: Kendinizi yeni bir projeye yönlendirmek
+  finding-a-project-to-contribute-to: Katkıda bulunacak bir proje bulma
+  how-to-submit-a-contribution: Nasıl katkı yapılır?
+  what-happens-after-you-submit-a-contribution: Bir katkı gönderdikten sonra ne olur?
+order: 1
+image: "/assets/images/cards/contribute.png"
+related:
+  - beginners
+  - building
+---
+
+## Açık kaynağa neden katkıda bulunur?
+
+<aside markdown="1" class="pquote"><img src="https://avatars.githubusercontent.com/errietta?s=180" class="pquote-avatar" alt="avatar"> \ [Freenode \] üzerinde çalışmak, daha sonra üniversitedeki çalışmalarımda ve gerçek işimde kullandığım becerilerin çoğunu kazanmama yardımcı oldu. Açık kaynak kodlu projeler üzerinde çalışmak projeye yardım ettiği kadar yapana da yardımcı olacağını düşünüyorum! <p markdown="1" class="pquote-credit"> - @errietta, ["Neden açık kaynaklı yazılıma katkıda bulunmayı seviyorum"] (https://www.errietta.me/blog/open-source/) </p></aside>
```

Comment:
It looks like many `<aside>`s have spurious backslashes and spaces added (presumably by a translation tool?), obtaining rendering like:

<img width="528" alt="Screen Shot 2019-09-28 at 13 37 18" src="https://user-images.githubusercontent.com/40415/65822233-2c6ba500-e1f5-11e9-836c-479e0a4bc2c3.png">

So I think changes like the suggestion below are needed throughout:

```suggestion
<aside markdown="1" class="pquote"><img src="https://avatars.githubusercontent.com/errietta?s=180" class="pquote-avatar" alt="avatar"> [Freenode] üzerinde çalışmak, daha sonra üniversitedeki çalışmalarımda ve gerçek işimde kullandığım becerilerin çoğunu kazanmama yardımcı oldu. Açık kaynak kodlu projeler üzerinde çalışmak projeye yardım ettiği kadar yapana da yardımcı olacağını düşünüyorum! <p markdown="1" class="pquote-credit"> - @errietta, ["Neden açık kaynaklı yazılıma katkıda bulunmayı seviyorum"](https://www.errietta.me/blog/open-source/) </p></aside>
```

Reply:
I used [GitLocalize](https://gitlocalize.com/) so It might have added the extra spaces and removed the newlines.

Reply:
Those two changes -- remove added space before `\` and restore newline before `<p markdown="1"` I think are needed throughout the articles.

Let me know if you can do that, otherwise I'll try some global search/replacing when I have time.

Reply:
Sorry for that. The tool makes translation easy in some way but I did not notice that it changed the markings or spaces. Let me try to fix it on my own.

Reply:
I have compared the article files with the master ones and tried to fix all the marking problems (additional spaces, lacking new lines etc). Can you please check on your side too?

Reply:
Thank you! Looks great, about to merge.